tests/test_commands.py

```diff
@@ -101,14 +101,27 @@ def run_bg(self, *command, script=None):
 
         yield proc
 
-        os.kill(proc.pid, signal.SIGINT)  # Ctrl+C
+        try:
+            # Shutdown the server via POST request
+            url = 'http://localhost:{}/__shutdown__/'.format(port)
+            print('Shutting down via', url)
+            requests.post(url)
+        except Exception as e:
+            print(e)
+            proc.kill()
+
```

Comment:
I'm getting an impression that we are taking the test tusite far more serious than we should :D